sklearn/ensemble/_hist_gradient_boosting/tests/test_gradient_boosting.py

```diff
@@ -744,3 +744,38 @@ def test_single_node_trees(Est):
 def test_custom_loss(Est, loss, X, y):
     est = Est(loss=loss, max_iter=20)
     est.fit(X, y)
+
+
+@pytest.mark.parametrize('HistGradientBoosting, X, y', [
+    (HistGradientBoostingClassifier, *_make_dumb_dataset(1200)),
+    (HistGradientBoostingRegressor, *_make_dumb_dataset(1200))
+])
+def test_staged_predict(HistGradientBoosting, X, y):
+
+    # Test whether staged predictor eventually gives
+    # the same prediction.
+    X_train, X_test, y_train, y_test = train_test_split(
+        X, y,
+        test_size=1000,
+        random_state=0
+    )
+    gb = HistGradientBoosting(
+        max_iter=10,
+        random_state=0
+    )
+    # test raise ValueError if not fitted
+    with pytest.raises(ValueError, match="..."):
```

Comment:
please write the error message so we know what's expected. You can just write part of it, no need to put everything

Reply:
I have changed to NotFittedError.